Documentation/PageTsconfig/TceMain.rst

```diff
@@ -286,32 +294,35 @@ group. The administrator does not have to set custom TSconfig to achieve this.
 
 This behaviour is similar to the "group sticky bit" in Unix for directories.
 
+..  index:: Page permissions; everybody
 
-.. index:: Page permissions; everybody
+..  _tcemain-permissions-everybody:
```

Comment:
```suggestion
..  _pagetcemain-permissions-everybody:
```

Above, "pagetcemain-permissions-copyFromParent:" is used for an anchor, please stay consistent.